autosynth/synth.py

```diff
@@ -225,19 +225,23 @@ def _inner_main(temp_dir: str) -> int:
         "--branch-suffix", default=os.environ.get("BRANCH_SUFFIX", None)
     )
     parser.add_argument("--pr-title", default="")
+    parser.add_argument("--default-branch", default=os.environ.get("DEFAULT_BRANCH"))
```

Comment:
With my suggested change to `change_pusher.py`, I think this argument becomes unnecessary.

Reply:
Done, we now always fetch the default branch. We still need the value so that we can send the pull request to the right base branch.